app/src/main/java/cm/aptoide/pt/home/apps/AppsAdapter.java

```diff
@@ -382,4 +400,20 @@ public void setAllUpdatesIndeterminate() {
       setAppStandby(((UpdateApp) app));
     }
   }
+
+  public void setAppOnPausing(App app) {
+    int indexOfApp = listOfApps.indexOf(app);
+    if (indexOfApp != -1) {
```

Comment:
Was this happening before or is this just defensive programming? Can the App that we are trying to pause not be in the view?

Reply:
This is just defensive programming. Since i am getting an index and that app might not be found, i decided to check for that. Same thing with the setStandby